javascript-loops/main.js

```diff
@@ -0,0 +1,87 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+var numbers = [];
+var currentNumbers = 1;
```

Comment:
You should not be making variables for use in a function outside of the function definition, that means these variables will continue to exist longer than they need to, which is not ideal. If you only intend to use a variable inside of a single function, that variable should be declared inside the function. Make sure you address this in your other function definitions as well.